go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -289,13 +290,87 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 		}
 	}
 
+	// create a reroute on the ovn_cluster_router
+	err = createLogicalRouterPolicyForNode(node, subnets)
+	if err != nil {
+		return fmt.Errorf("failed to create logical router policy: %v", err)
+	}
+
+	return nil
+}
+
+func createLogicalRouterPolicyForNode(node *kapi.Node, subnets []*net.IPNet) error {
+	// get all the windows subnets as the dest for lr-policy match
+	var dest string
+	var nextHop string
+	var src string
+	for _, hybridSubnet := range config.HybridOverlay.ClusterSubnets {
+		if !utilnet.IsIPv6CIDR(hybridSubnet.CIDR) {
```

Comment:
I'm guessing we are going to support ipv6/dual stack later?

Reply:
windows does not support ipv6 and will not for some time. As soon as we get word that windows supports ipv6 we can move forward with hybrid ipv6/dual stack

Reply:
@alexanderConstantinescu hybridOverlayExternalGW is not related to Windows and Services that this PR touches; it's instead the v1 of the F5 connector. So there is no conflict.